go-controller/pkg/cni/helper_linux.go

```diff
@@ -101,20 +104,33 @@ func setupInterface(netns ns.NetNS, containerID, ifName, macAddress, ipAddress,
 }
 
 // ConfigureInterface sets up the container interface
-func (pr *PodRequest) ConfigureInterface(namespace string, podName string, macAddress string, ipAddress string, gatewayIP string, mtu int, ingress, egress int64) ([]*current.Interface, error) {
+// TODO alona update the window_helper
+func (pr *PodRequest) ConfigureInterface(namespace string, podName string, networkName string, macAddress string, ipAddress string, gatewayIP string, mtu int, ingress, egress int64) ([]*current.Interface, error) {
 	netns, err := ns.GetNS(pr.Netns)
 	if err != nil {
 		return nil, fmt.Errorf("failed to open netns %q: %v", pr.Netns, err)
 	}
 	defer netns.Close()
 
-	hostIface, contIface, err := setupInterface(netns, pr.SandboxID, pr.IfName, macAddress, ipAddress, gatewayIP, mtu)
+	isDefaultInterface := networkName == "ovn-kubernetes"
+	var ifaceID string
+	var hostIfaceName string
+	if isDefaultInterface {
+		ifaceID = fmt.Sprintf("%s_%s", namespace, podName)
+		hostIfaceName = pr.SandboxID[:15]
```

Comment:
this is likely to exceed 15 chars. IMO, we should not silently truncate as different pods may step on one another. some options:
 - if the name is longer than 15 chars, submit an error (not so good, as it will require changes to pod names and/or namespaces)
 - use a hash function instead of truncation, makes collisions very unlikely (problem is that names are becoming meaningless)

Reply:
The sandbox id is not a name, it is more like a uuid.
But anyway, this is the old code for the default pod network. I don't want to change it in the scope of this PR.